policyengine_us/parameters/gov/states/mo/tax/income/credits/non_refundable.yaml

```diff
@@ -4,7 +4,7 @@ values:
   - mo_wftc
```

Comment:
This credit comes into effect in 2023 right? (Based on the formula_2023), if so we need to do the following:

2020-01-01: []

2023-01-01:
    -mo_wftc